kuwala/pipelines/population-density/README.md

```diff
@@ -26,7 +26,7 @@ For more general information check out our
 ## Run
 
 Proceed only if you followed the initial steps to initialize the main components mentioned here:
-[`Initialize Main Components`](https://github.com/kuwala-io/kuwala/tree/master/kuwala-pipelines/)
+[`Initialize Main Components`](https://github.com/kuwala-io/kuwala/tree/master/kuwala/)
```

Comment:
The path for the image in the docs folder needs to be adjusted as well (line 8).